libcontainer/intelrdt/intelrdt.go

```diff
@@ -144,6 +166,23 @@ import (
  *         "memBwSchema": "MB:0=20;1=70"
  * 	}
  * }
+ *
+ * Another example:
+ * We only want to monitor memory bandwidth and llc occupancy.
+ * "linux": {
+ *     "intelRdt": {
+ * 	}
+ * }
+ *
+ * or
+ *
+ * "linux": {
+ *     "intelRdt": {
+ *         "l3CacheSchema": "",
+ *         "memBwSchema": ""
+ * 	}
+ * }
+ *
```

Comment:
When the config interface is determined, make the same change in libcontainer/SPEC.md